walletlibrary/src/main/java/com/microsoft/did/sdk/credential/service/protectors/IssuanceResponseFormatter.kt

```diff
@@ -0,0 +1,119 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+package com.microsoft.did.sdk.credential.service.protectors
+
+import android.util.Base64
+import com.microsoft.did.sdk.credential.service.*
+import com.microsoft.did.sdk.credential.service.models.oidc.AttestationClaimModel
+import com.microsoft.did.sdk.credential.service.models.oidc.IssuanceResponseClaims
+import com.microsoft.did.sdk.crypto.CryptoOperations
+import com.microsoft.did.sdk.crypto.DigestAlgorithm
+import com.microsoft.did.sdk.crypto.keyStore.EncryptedKeyStore
+import com.microsoft.did.sdk.identifier.models.Identifier
+import kotlinx.serialization.json.Json
+import java.util.*
```

Comment:
Diff also shows this was explicitly `java.util.UUID` (across a number of files).